Library/ViewModels/ShippingRulesViewModel.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol ShippingRulesViewModelInputs {
+  func configureWith(_ project: Project, shippingRules: [ShippingRule], selectedShippingRule: ShippingRule)
+  func viewDidLoad()
+}
+
+public protocol ShippingRulesViewModelOutputs {
+  var values: Signal<[String], Never> { get }
+}
+
+public protocol ShippingRulesViewModelType {
+  var inputs: ShippingRulesViewModelInputs { get }
+  var outputs: ShippingRulesViewModelOutputs { get }
+}
+
+public final class ShippingRulesViewModel: ShippingRulesViewModelType,
+  ShippingRulesViewModelInputs, ShippingRulesViewModelOutputs {
+  public init() {
+    self.values = self.configDataProperty.signal
+      .takeWhen(self.viewDidLoadProperty.signal)
```

Comment:
So this is cool to do, just worth noting that in all the other areas we've used `combineLatest` because then we aren't dependent on the order in which these inputs are called. Using `takeWhen` just means that we _must_ call the `configureWith()` before `viewDidLoad()`. In this case it's fine, just wanted to point that out.

Reply:
Oh, that's good to know! Maybe I can change this so that it doesn't bite us later?

Reply:
It may be safer and more consistent to use `combineLatest`, yeah.